specs/eventing/broker.md

```diff
@@ -145,6 +145,18 @@ The subscriber MAY receive a confirmation that a reply event was accepted by the
 Broker. If the reply event was not accepted, the initial event SHOULD be
 redelivered to the subscriber.
 
+Subscribers SHOULD acknowledge delivery of an event with an HTTP 2xx response code.
```

Comment:
I think if you reword this we can turn this into a MUST. “In the event of a successful delivery, the subscriber MUST respond with an HTTP 2xx response code”

Reply:
I kept this as `SHOULD` based on @n3wscott's previous comment - assuming these are recommendations. 

>We have not been this strict on the exact response code, accepting 2xx means ACK, but we could have a MAY section that says here is what a 204 and a 200 could mean.

I can see how this intro sentence could be `MUST` in that successful delivery necessitates a 2xx code.

Reply:
Yup - I think having a discussion around which 2xx to return when is ok, but I think we need to mandate (MUST) that a successful delivery see some kind of  2xx. Otherwise, with a "SHOULD" someone could be spec compliant but return 500 for everything.

Reply:
OK, I see your point.If this is changed to `MUST`
```suggestion
Subscribers MUST acknowledge successful delivery of an event with an HTTP 2xx response code.
```
Then all of the remaining 2xx can remain `MAY`. You are correct, these response code requirements are on the sink, not the broker, so really any additional contextual meaning is ignored by Knative. I just made the assumption that the broker will accept and route any CloudEvent in a response from a sink and route it with any `2xx`. That's based on [this](https://github.com/knative/specs/blame/main/specs/eventing/broker.md#L138-L140).

> Events contained in delivery responses SHOULD be published to the Broker ingress and processed as if the event had been produced to the Broker's addressable endpoint.

Should I include some text that makes it clear these response codes are recommendations for event receivers, but that currently there are no requirements on the broker to act in any special way based on these `2xx` response codes. Any successful response will do?